examples/classification.ipynb

```diff
@@ -5,7 +5,11 @@
    "id": "cb9e5d62",
```

Comment:
<p>List markdown is broken</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>It renders fine on github and locally.</p>

Reply:
<p>Ok, so let's leave it as is, I guess.</p>